CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 
 * **[Fix]** Fix naming conflict with iOS 14 private Apple framework.
 
+#### Android/iOS
```

Comment:
Not sure that it should be in `Android/iOS` section, because these changes will be called only when you start your app in Unity Editor.

Reply:
Done.